### PR TITLE
Try solving lex syntax error

### PR DESCRIPTION
**Pull request by szepeviktor**
> ©

SyntaxError: Non-ASCII character '\xc2' in file setup.py on line 6, but no encoding declared

### CODE REVIEW

**Comment by szepeviktor** [comment]
@Nikratio Why run pdflatex **three** times?

`for _ in range(3):`

**Comment by szepeviktor** [comment]
Seems like it is all green now.

If you have time please fix tex warnings: https://travis-ci.org/s3ql/s3ql/jobs/266480277
And decide on *Running pdflatex* related changes.

**Comment by Nikratio** [comment]
Having to run LaTeX several times unfortunately intrinsically required by the way LaTeX works. For example, the first run generates files that are used in the second run to generate the table of contents (which will be empty in the first set of files). The third run is required because inserting the ToC will change page numbers, so the ToC in the second run will be complete but have wrong page numbers.

**Comment by Nikratio** [comment]
The sphinx warnings seem to be caused by running Sphinx several times. Apparently each run tries to re-register the extensions defined in `conf.py`. So the warning is harmless. But I agree that this should be fixed. I'll open a separate bug about that.

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ python:
 cache: pip
 
 before_install:
+  - sudo add-apt-repository -y ppa:jonathonf/texlive-2016
```
Comment (Nikratio) [inline .travis.yml]: Could you clarify why this is needed / what error is fixed by this? Thanks!

Comment (szepeviktor) [inline .travis.yml]: First I thought the origin of errors is the old (2013) version of texlive.
Then it was left there to use modern software.
What is your texlive version?

Comment (Nikratio) [inline .travis.yml]: I'm using 2016 as well, but I don't think the files use any recent features. *Which* errors do you mean?

**Comment by Nikratio** [comment]
I've merged the encoding change and the split into separate build commands in `.travis.yml`.

File: .travis.yml (hunk 1 above)
Comment (szepeviktor) [inline .travis.yml]: The red ones, e.g.

> /home/travis/build/s3ql/s3ql/rst/man/ctrl.rst:60: WARNING: Could not lex literal_block as "commandline". Highlighting skipped.

https://travis-ci.org/s3ql/s3ql/jobs/266480277

Comment (szepeviktor) [inline .travis.yml]: I mean it is *to be fixed* - not fixed by using this repo.

Should I revert ppa:jonathonf/texlive-2016?

Comment (Nikratio) [reply .travis.yml]: These are Sphinx errors, so they're not going to be affected by updating texlive.

Comment (szepeviktor) [reply .travis.yml]: OK!

**Pull request closed without merging**